.github/workflows/eks-performance-cluster-scaling.yml

```diff
@@ -105,6 +132,18 @@ jobs:
         if: github.event_name == 'workflow_dispatch'
         run: |
           echo "Starting manual scaling operation with desired capacity: $DESIRED_CAPACITY_PER_NODEGROUP"
+          echo "Leader node desired capacity: $LEADER_NODE_DESIRED_CAPACITY"
```

Comment:
does the current leader model support more than 1 instance? Im just curious what scenario we would want to test more than 1 leader case.